clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -0,0 +1,446 @@
+//==---- group_utils.h ------------------*- C++ -*--------------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------===//
+
+#ifndef __DPCT_GROUP_UTILS_HPP__
+#define __DPCT_GROUP_UTILS_HPP__
+
+#include <stdexcept>
+#include <sycl/sycl.hpp>
+
+#ifdef SYCL_EXT_ONEAPI_USER_DEFINED_REDUCTIONS
+#include <sycl/ext/oneapi/experimental/user_defined_reductions.hpp>
+#endif
+
+#include "dpct/dpct.hpp"
+#include "dpct/dpl_extras/functional.h"

```

Comment:
```suggestion
#include "dpct.hpp"
#include "dpl_extras/functional.h"

```
I believe you have the relative dir incorrect here.

Reply:
Yes I had this earlier, but some how that was giving errors as well. I will re -apply these changes.